qrenderdoc/Code/QRDUtils.cpp

```diff
@@ -480,6 +480,36 @@ void RichResourceTextInitialise(QVariant &var, ICaptureContext *ctx)
 
     linkedText->ctxptr = ctx;
 
+    // convert URL into clickable links
+    auto handleURLFragment = [&linkedText](QString str) {
+      static QRegularExpression urlRE(lit(
+          R"(https://[www.]?[-a-zA-Z0-9@:%._+~#=]{2,256}.[a-z]{2,4}\b[-a-zA-Z0-9@:%_+.~#?&/=]*)"));
```

Comment:
URL matching with a regex is complex and not necessary for us, but I think this could be improved a bit:

```suggestion
          R"(https?://([a-zA-Z0-9]+\.)?[-a-zA-Z0-9@:%._+~#=]{2,256}.([a-z]{2,8})+/[-a-zA-Z0-9@:%_+.~#?&/=]*)"));
```

Allow http links, support any word-like subdomain rather than just www, and support nested TLDs like `.co.uk`.

That way it covers a bit wider array of URLs.